<launch>
    <group ns="ahrs">
        <node name="ahrs_node" pkg="ahrs" type="ahrs_node" respawn="false" />
        <node name="tf_boradcaster" pkg="ahrs" type="tf_broadcaster.py" respawn="false" />
    </group>
</launch>
